src/main/java/org/folio/rest/impl/VoucherHelper.java

```diff
@@ -84,14 +87,16 @@ public Future<VoucherCollection> getVouchers(String query, int offset, int limit
    * @return completable future holding response indicating success or error if failed
    */
   public Future<Void> partialVoucherUpdate(String id, Voucher voucher, RequestContext requestContext) {
-    return voucherService.partialVoucherUpdate(id, voucher, requestContext);
+    return voucherService.partialVoucherUpdate(id, voucher, requestContext)
```

Comment:
I see that voucher has many similar fields that located in invoice object, for example accounting code, accountNu, disbursementNumber etc.
Thinking do we need to update them as well, not only voucherNumber.
Is it possible to edit some voucher fields instead of voucher number? If not - probably it's enought to update only voucher number.

Reply:
sure, I will check on that. thank you

Reply:
I checked on the Invoice part. we can only edit the voucher number from invoice. so probably, that is all we can do about voucher from invoice.